src/Microsoft.AspNet.OData.Shared/Builder/BuilderHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNet.OData.Common;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNet.OData.Builder
+{
+    /// <summary>
+    /// Helper Method for Builder
+    /// </summary>
+    public static class BuilderHelper
```

Comment:
1) don't need this new class, you can move it  TypeHelper.cs .
don't make it public